docs/snippets/common/storybook-theme-example-variables.ts.mdx

```diff
@@ -8,5 +8,6 @@ export default create({
   brandTitle: 'My custom storybook',
   brandUrl: 'https://example.com',
   brandImage: 'https://place-hold.it/350x150',
+  brandTarget: '_self',
```

Comment:
There's a bulleted list below this snippet on the page in which it's embedded (https://github.com/storybookjs/storybook/blob/next/docs/configure/theming.md), which would also need updated to explain what this line does.

Reply:
I guess also fixed

Reply:
Thanks! I added a commit to update the format of the content to match the existing content.